code/game/gamemodes/meme/meme.dm

```diff
@@ -1,181 +0,0 @@
-//This file was auto-corrected by findeclaration.exe on 29/05/2012 15:03:04
-
-/datum/game_mode/var/list/memes = list()
-
-/datum/game_mode/meme
-	name = "Memetic Anomaly"
-	config_tag = "meme"

```

Comment:
Now I can tell players in the forums thread we really did cut down on the amount of memes in the codebase!